lib/tx.js

```diff
@@ -7,7 +7,8 @@ var AuthPollStopError = require('./errors/AuthPollStopError');
 var config            = require('./config');
 
 function addStateToken(res, options) {
-  var builtArgs = util.clone(options) || {};
+  var builtArgs = {};
+  util.extend(builtArgs, options || {});
```

Comment:
Changed this to use extend. clone uses JSON.stringify which makes it difficult to use with functions.

Reply:
This _should_ work without the `|| {}` check